rest/index.py

```diff
@@ -72,6 +72,21 @@ def status():
     return response
 
 
+# placeholder for future watchlist request #
+@route('/getwatchlist')
+@route('/getwatchlist/')
+def status():
+    '''endpoint for grabbing watchlist contents'''
+    if request.body:
+        request.body.read()
+        request.body.close()
+    response.status = 200
+    response.content_type = "application/json"
+    response.body = getWatchlist()
+    sendMessgeToPlugins(request, response, 'getwatchlist')
```

Comment:
This 'sendMessageToPlugins' line isn't needed, as we don't actually have a plugin that registers for "getwatchlist". I believe you can remove the line entirely.

Reply:
It would fit the pattern better if the thing that makes up the watchlist is actually a plugin that could be custom to one's environment, but gets called via the rest interface. i.e. like the blocklist, or logincounts, etc. The endpoint is generic but the plugin has the business logic/tech integration/etc.

Reply:
oh, this is the 'get' endpoint.. /me reads more

Reply:
Still probably a good idea to pull the getWatchlist() function out into a plugin. That would give folks who don't want the UI a way to still use the watchlist feature fed by something else.

Reply:
We have a way to use this without the UI, it's called generic alerts. 
The point of this is to have the ability to quickly add a watchlist item through the UI without having to create PR or wait for a code update.